simplified-accounts-registry-api/src/main/java/org/nypl/simplified/accounts/registry/api/AccountProviderRegistryType.kt

```diff
@@ -42,9 +42,12 @@ interface AccountProviderRegistryType {
 
   /**
    * Refresh the available account providers from all sources.
+   *
+   * @param includeTestingLibraries A hint for providers indicating whether
+   * testing libraries should be loaded. May be ignored by some providers.
    */
 
-  fun refresh()
+  fun refresh(includeTestingLibraries: Boolean)
```

Comment:
This is fine for now, but we'll need a more general mechanism than this shortly to handle things like "libraries near me" requests. Yes, that subject again :smile:

Reply:
Yeah, I completely agree, it's not super flexible.